.github/workflows/cloudbeat-ci.yml

```diff
@@ -20,9 +20,8 @@ jobs:
       - name: Check out the repo
         uses: actions/checkout@v2
         with:
-          token:           ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This checkout def works, not sure regarding the change.
Would be better to keep it as is.